features/config/mock/src/main/java/org/opennms/netmgt/dao/mock/ConfigurationManagerServiceMock.java

```diff
@@ -86,60 +92,183 @@ public void changeConfigDefinition(String configName, ConfigDefinition configDef
         if ("provisiond".equals(configName)) {
             def = XsdHelper.buildConfigDefinition("provisiond", "provisiond-configuration.xsd",
                     "provisiond-configuration", ConfigurationManagerService.BASE_PATH);
+        } else if ("enlinkd".equals(configName)) {
```

Comment:
I think the best one may be able to read the changelog file. But it may take too long time to finish that. Lets unblock CM2 first and think about it later.